rest_framework_smoke/tests/mixins.py

```diff
@@ -153,7 +153,7 @@ def get_list_schema(self, min_items: int = 1) -> dict:
 
         schema = self.pagination_schema
         schema.update({"results": result_list_schema})
-        return schemas.get_object_schema(schema)
+        return schema
```

Comment:
Relaxing schema constraits is reasonable for paginaged list responses, as it's difficult to prepare large fixtures to fill next/previous links, but instead there should be a mixin with some tests that check next/previous links in paginates response